ubuntudesign/vanilla-framework
# Converted the brand color to light grey with some affordances for light ...

...brand colours

# Done
- Converted the brand color of vanilla from ubuntu orange to light grey
- Added header link color var to set the header link colours
- Changed the site logo from white to orange
- Removed some extra tabbing in global settings
- Button retains the ubuntu orange

## QA 
- Run `gulp build`
- Run `google-chrome demo/index.html` and check the demo header is light grey and looks ok?
